lib/src/internal/copy_on_write_list.dart

```diff
@@ -19,12 +19,26 @@ class CopyOnWriteList<E> implements List<E> {
   @override
   E operator [](int index) => _list[index];
 
+  @override
+  // TODO: Dart 2.0 requires this method to be implemented.
+  // ignore: override_on_non_overriding_method
+  List<E> operator +(List<E> other) {
+    throw new UnimplementedError("+");
```

Comment:
Please use single quotes for strings; everywhere else, too :/

Reply:
Done!